app/src/main/java/com/android/unio/model/events/EventRepositoryFirestore.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.unio.model.events
+
+import com.google.firebase.firestore.FirebaseFirestore
+
+class EventRepositoryFirestore : EventRepository {
+  private val firestore = FirebaseFirestore.getInstance()
+
+  // Change the return type to accept a callback
```

Comment:
ChatGPT ?

Reply:
This one will be destroyed, as the other for the same explanation